src/generators/bendable-amogus/BendableAmogus.res

```diff
@@ -0,0 +1,239 @@
+let requireImage = fileName => Generator.requireImage("./images/" ++ fileName)
+let requireTexture = fileName => Generator.requireImage("./textures/" ++ fileName)
+
+let id = "bendableamogusgenerator"
+
+let name = "Bendable Amogus generator"
+
+let images: array<Generator.imageDef> = [
+  {id: "Background", url: requireImage("Background.png")},
+  {id: "Folds", url: requireImage("Folds.png")},
+  
+]
+
+let textures: array<Generator.textureDef> = [
+  {
+    id: "Skin",
+    url: requireTexture("Skin.png"),
+    standardWidth: 64,
+    standardHeight: 64,
+  },
+  {
+    id: "Colors",
+    url: requireTexture("Colors.png"),
+    standardWidth: 16,
+    standardHeight: 2,
+  },
+]
+
+let script = () => {
+  Generator.defineSelectInput("Color", [
+        "Red",
+        "Black",
+        "White",
+        "Rose",
+        "Blue",
+        "Cyan",
+        "Yellow",
+        "Pink",
+        "Purple",
+        "Orange",
+        "Banana",
+        "Coral",
+        "Lime",
+        "Green",
+        "Gray",
+        "Maroon",
+        "Brown",
+        "Tan",
+        
+      ]);
+
+      Generator.defineTextureInput("Skin", {
+        standardWidth: 64,
+        standardHeight: 64,
+        choices: []
+      });
+      Generator.drawImage("Background", (0, 0))
+  let red = Generator.getSelectInputValue("Color") === "Red";
+let black = Generator.getSelectInputValue("Color") === "Black";
+let white = Generator.getSelectInputValue("Color") === "White";
+let rose = Generator.getSelectInputValue("Color") === "Rose";
+let blue = Generator.getSelectInputValue("Color") === "Blue";
+let cyan = Generator.getSelectInputValue("Color") === "Cyan";
+let yellow = Generator.getSelectInputValue("Color") === "Yellow";
+let pink = Generator.getSelectInputValue("Color") === "Pink";
+let purple = Generator.getSelectInputValue("Color") === "Purple";
+let orange = Generator.getSelectInputValue("Color") === "Orange";
+let banana = Generator.getSelectInputValue("Color") === "Banana";
+let coral = Generator.getSelectInputValue("Color") === "Coral";
+let lime = Generator.getSelectInputValue("Color") === "Lime";
+let green = Generator.getSelectInputValue("Color") === "Green";
+let gray = Generator.getSelectInputValue("Color") === "Gray";
+let maroon = Generator.getSelectInputValue("Color") === "Maroon";
+let brown = Generator.getSelectInputValue("Color") === "Brown";
+let tancolour = Generator.getSelectInputValue("Color") === "Tan";
+
+
+  
+if red {
+  Generator.drawTextureLegacy(
+    "Colors",
+    {x: 0, y: 0, w: 1, h: 1},
+    {x: 0, y: 0, w: 600, h: 660},
+    (),
+  ) };
```

Comment:
If you like, there is a slightly less verbose syntax you can use here for the rectangles.

Instead of the curly bracket syntax `{x: 0, y: 0, w: 1, h: 1}`, you can use the round bracket syntax `(0, 0, 1, 1)`.

So this will look like:


```
Generator.drawTexture(
  "Colors",
  (0, 0, 1, 1),
  (0, 0, 600, 660),
  ()
)
```

Whichever you prefer is fine.